Roles/Firewall/tests/test.yml

```diff
@@ -0,0 +1,5 @@
+---
+- hosts: localhost
+  remote_user: root
+  roles:
+    - Firewall
```

Comment:
auto edw to kommati den thumamai na to ekana kan! den to exw shmeiwsei kan sto shmeiwmatario pou krataw thn methodologia pou akolouthw gia na mhn ta ksexasw.... ti mporei na einai? skeftomai na to svhsw